docs/src/pages/customization/typography/ResponsiveFontSizesChart.js

```diff
@@ -1,4 +1,4 @@
-/* eslint-disable material-ui/restricted-path-imports */
+/* eslint-disable no-restricted-imports */
```

Comment:
Actually works.

@oliviertassinari What's the plan for `cssUtils`? It's not that bad to use the path import since module duplication or bundle size are negligible. Would still be nice to have a consistent, simple recommendation.

Reply:
The `cssUtils` is used for a private demo (the source isn't exposed to the users). I'm not sure to follow your question. What do you mean by plan?

Reply:
> The cssUtils is used for a private demo (the source isn't exposed to the users).

Right. I thought this was a public demo. I'll add a comment explaining this.

Reply:
Added an explainer:
```js
// import of a small, pure module in a private demo
// bundle size and module duplication is negligible
```